src/NewProject/new_project_dialog.h

```diff
@@ -75,6 +75,8 @@ class newProjectDialog : public QDialog {
   void UpdateDialogView(Mode mode = NewProject);
   void ResetToNewProject();
   void ResetToProjectSettings();
+  bool ValuesValid(QString* errStr = nullptr);
```

Comment:
const would be good here

Reply:
Fixed, thanks for the reminder, i'll eventually remember to check for this ;)